src/ui/popupmenu/popupmenu.rs

```diff
@@ -388,9 +389,11 @@ impl Popupmenu {
                 prev.info.set_visible(false);
                 prev.menu.set_visible(false);
 
-                // Update the `kind` icon with default fg color.
+                // Update the `kind` icon with default fg color if it is an icon.
                 let buf = get_icon_pixbuf(&prev.item.kind, &fg, font_height);
```

Comment:
Move this inside the if clause. Also revert the documentation to the original form.